package.json

```diff
@@ -93,8 +92,8 @@
     "@babel/plugin-proposal-object-rest-spread": "^7.12.1",
     "@babel/polyfill": "7.4.3",
     "@date-io/moment": "^1.3.13",
-    "@material-ui/core": "^3.9.4",
-    "@material-ui/icons": "^4.9.1",
+    "@material-ui/core": "^3.0.0",
```

Comment:
Please document whether the material-ui core needs to revert from 3.9.4 (2 years old) to 3.0.0 (3 years old, a version no-one uses), and whether material-ui icons needs to revert from 4.9.1 (2 years old) to 3.0.2 (3 years old).  I don't want to revert to older versions unless absolutely necessary